components/navbar.vue

```diff
@@ -1,157 +1,193 @@
 <template>
-  <nav>
-    <div
+  <div id="wrapper">
+    <button
+      id="toggle"
       class="
+        navbar-toggle
+        bg-transparent
+        select-none
+        border-none
+        focus:outline-none
```

Comment:
you did like the user to have some form of a visual idea that click has happened for accessibility reasons. You can do one thing instead of ugly looking outside border make the border or outline inside of button might look better

Reply:
Actually the Click will initiate the button to transform into a Cross button and vice versa.

Reply:
Alright then it's cool